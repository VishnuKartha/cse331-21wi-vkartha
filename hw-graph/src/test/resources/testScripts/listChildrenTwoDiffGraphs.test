# listing the children of two different graphs

# Create a Graph
CreateGraph graph1
CreateGraph graph2

# Add a pair of nodes
AddNode graph1 A
AddNode graph1 B
AddNode graph2 C
AddNode graph2 D

# Adds 2 edges
AddEdge graph1 A B e1
AddEdge graph2 C D e2

# list the children of both graphs
ListChildren graph1 A
ListChildren graph2 C
