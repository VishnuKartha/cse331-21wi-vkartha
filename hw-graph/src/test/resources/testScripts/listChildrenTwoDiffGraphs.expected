# listing the children of two different graphs

# Create a Graph
created graph graph1
created graph graph2

# Add a pair of nodes
added node A to graph1
added node B to graph1
added node C to graph2
added node D to graph2

# Adds 2 edges
added edge e1 from A to B in graph1
added edge e2 from C to D in graph2

# list the children of both graphs
the children of A in graph1 are: B(e1)
the children of C in graph2 are: D(e2)